pkgs/data/fonts/windows-fonts/default.nix

```diff
@@ -0,0 +1,101 @@
+{
+  lib,
+  stdenvNoCC,
+  fetchurl,
+  p7zip,
+}:
+let
+  windows-fonts =
+    {
+      pname,
+      version,
+      url,
+      sha256,
+      desc,
+    }:
+    let
+      languageFonts = import ./${pname}-languages.nix;
+      source =
+        with lib.attrsets;
+        mapAttrs' (
+          name: fonts:
+          nameValuePair (name) (builtins.concatStringsSep " " (map (font: "Windows/Fonts/" + font) fonts))
+        ) languageFonts;
+    in
+    stdenvNoCC.mkDerivation {
+      inherit pname version desc;
+
+      src = fetchurl { inherit url sha256; };
+
+      outputs = [ "out" ] ++ builtins.attrNames source;
+
+      nativeBuildInputs = [ p7zip ];
+
+      unpackPhase = ''
+        runHook preUnpack
+        7z x $src
+        7z x -aoa sources/install.wim Windows/{Fonts/"*".{ttf,ttc},System32/Licenses/neutral/"*"/"*"/license.rtf}
+        runHook postUnpack
+      '';
+
+      installPhase =
+        ''
+          runHook preInstall
+
+          mkdir -p $out/share/fonts/truetype
+        ''
+        + lib.strings.concatLines (
+          lib.lists.forEach (builtins.attrNames source) (
+            name:
+            (
+              ''install -Dm644 ${source.${name}} -t $''
+              + name
+              + ''
+                /share/fonts/truetype
+                              ln -s $''
+              + name
+              + ''
+                /share/fonts/truetype/*.{ttf,ttc} $out/share/fonts/truetype
+                              install -Dm644 Windows/System32/Licenses/neutral/*/*/license.rtf -t $''
+              + name
+              + ''
+                /share/licenses
+              ''
```

Comment:
My source code is this:

```nix
''install -Dm644 ${source.${name}} -t $'' + name + ''/share/fonts/truetype
  ln -s $'' + name + ''/share/fonts/truetype/*.{ttf,ttc} $out/share/fonts/truetype
  install -Dm644 Windows/System32/Licenses/neutral/*/*/license.rtf -t $'' + name + '' /share/licenses
''
```

Because it needs to go through nixfmt-rfc-style, it becomes commit like this, I tried to change it, it should be much better if I declare a variable.

Current Code:
```nix
let
  outHome = "$" + name;
in
(''
  install -Dm644 ${source.${name}} -t ${outHome}/share/fonts/truetype
  ln -s ${outHome}/share/fonts/truetype/*.{ttf,ttc} $out/share/fonts/truetype
  install -Dm644 Windows/System32/Licenses/neutral/*/*/license.rtf -t ${outHome}/share/licenses
'')
```